slack-base/src/main/java/com/hubspot/slack/client/models/blocks/messages/RichTextListIF.java

```diff
@@ -0,0 +1,24 @@
+package com.hubspot.slack.client.models.blocks.messages;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@HubSpotStyle
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public interface RichTextListIF extends RichMessageBlock {
+    String TYPE = "rich_text_list";
+    @Override
+    @Value.Derived
+    default String getType() {
+        return TYPE;
+    }
+
+    RichBlockStyle getStyle();
+
+    int getIndent();
+
+    int getBorder();
```

Comment:
What does it represent? I'm curious about the difference between 1 and 2 for example.

Reply:
Also, it's good you asked about this, I've added the optional offset field, which is present in a few `RichTextList` blocks, but not in all of them.